File: src/main/java/com/wlyscraft/slimefunitemexpansion/tasks/ResourceRegisterTask.java
package com.wlyscraft.slimefunitemexpansion.tasks;

import com.wlyscraft.slimefunitemexpansion.SlimefunItemExpansion;
import com.wlyscraft.slimefunitemexpansion.items.IEItems;
import com.wlyscraft.slimefunitemexpansion.resources.OsmiumDust;

public class ResourceRegisterTask {
    public static void run(){
        new OsmiumDust(IEItems.IE_RESOURCE_STUFF).register(SlimefunItemExpansion.getInstance());
    }
}

File: src/main/kotlin/io/github/itsflicker/slimefunitemexpansion/tasks/ResourceRegisterTask.kt
package io.github.itsflicker.slimefunitemexpansion.tasks

import io.github.itsflicker.slimefunitemexpansion.items.IEItems
import io.github.itsflicker.slimefunitemexpansion.items.resources.OsmiumDust

object ResourceRegisterTask {

    fun run() {
        OsmiumDust(IEItems.IE_RESOURCE_STUFF).register(io.github.itsflicker.slimefunitemexpansion.SlimefunItemExpansion)
    }
}